Operations_C/main.c

```diff
@@ -39,17 +40,93 @@ int Compare(int a, int b)
     else return -1;
 }
 
+void MoveBites()
+{
+    unsigned char x = 0b00010010;
+
+    unsigned char n = x << 3;
+
+    unsigned char m = n >> 2;
+
+    printf("Number before the move is %d\n", x);
+
+    printf("Number after triple left shift is %d\n", n);
+
+    printf("Number after double right shift is %d\n", m);
+}
+
+int XOR (char a, char b)
+{
+    if (a != 0)
+    {
+        a = 1;
+    }
+
+    if (b != 0)
+    {
+        b = 1;
+    }
+
+    char c = a ^ b;
+
+    printf("XOR of these variables can be 1 - true or 0 - false. Result: %d\n", c);
+
+    return 0;
+}
+
+void PrintBites(int x)
+{
+    int firstLoopVar = x;
+
+    int cellCounter = 0;
+
+    while (firstLoopVar > 0)
+    {
+        firstLoopVar = firstLoopVar / 2;
+
+        cellCounter++;
+    }
+
+    int Binary[cellCounter];
```

Comment:
Even that you can define your variable whenever you want, by convention it is better to define Function scope variables in the beginning of the method. Please move defining of this method to the beginning.

Reply:
I wanted to delete one loop which determined how many members the array has and did it, but i created new one to make all 32 members of the array to 0.